app/konnect/dev-portal/administrators/public-portal.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Public Developer Portal
+no_version: true
+---
+
+Enable public access to your Dev Portal. When Public Portal is enabled, developers
+don't need to register for the portal access and any new application registrations aren't
```

Comment:
> new application registrations aren't allowed

Is there any security risk with existing application registrations? Or even if not, maybe this is something the admin should know: enabling public access means that app reg is disabled, but any existing registered applications will continue to work.

Reply:
> > new application registrations aren't allowed
> 
> Is there any security risk with existing application registrations? Or even if not, maybe this is something the admin should know: enabling public access means that app reg is disabled, but any existing registered applications will continue to work.

I'm not sure if there are any security risks. @skibum would have to answer that question. Also it seems confusing to say app reg is disabled when the button can stay enabled!

Reply:
There is not a security risk for previously approved applications.

Reply:
Thanks for your response @skibum.